packages/eslint-plugin-material-ui/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "eslint-plugin-material-ui",
+  "version": "1.0.0",
+  "description": "Custom eslint rules for Material-UI",
+  "repository": "https://github.com/callemall/material-ui/packages/eslint-pluigin-material-ui",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "../../node_modules/mocha/bin/_mocha -- tests/lib/**/*.js",
```

Comment:
What happens if you just use `mocha` and `eslint` for these instead of the relative paths to the parent node_modules folder? does it work or break?